mountpoint-s3/tests/fuse_tests/unlink_test.rs

```diff
@@ -143,7 +146,10 @@ where
     );
 
     drop(f); // close file
-
+             // when there are multiple tests called release method can be called later than unlink or rmdir.
+             // So, adding the sleep to ensure release being called earlier.
+    let result = test_client.contains_key("dir/writing.txt");
+    sleep_till_retry_succeed!(result);
```

Comment:
Fix indentation here. Also the message can be a little clearer.

```suggestion
    // `release` operation triggered by `drop(File)` is asynchronous.
    // Add sleep to ensure condition is checked after `release` completes.
    let result = test_client.contains_key("dir/writing.txt");
    sleep_till_retry_succeed!(result);
```